src/commands/scanner/run.ts

```diff
@@ -87,7 +87,7 @@ export default class Run extends ScannerCommand {
 			char: 'f',
 			description: messages.getMessage('flags.formatDescription'),
 			longDescription: messages.getMessage('flags.formatDescriptionLong'),
```

Comment:
I reordered the options to be in alphabetical order, done in a couple of places.